src/routes/project.ts

```diff
@@ -0,0 +1,38 @@
+import { Router } from "express";
+import ProjectController from "../controllers/ProjectController";
+import { checkJwt } from "../middlewares/checkJwt";
```

Comment:
I think It could be better if the file name is auth.middleware.ts and the function name is authMiddleware